data/blog/id-headers-tailscale-serve-flask.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Tapping into Tailscale's identity headers with serve
+date: 2023-07-19
+authors: ['parker', 'sonia']
+tags:
+  - serve
+  - headers
+  - python
+  - flask
+  - funnel
+summary: When you use Tailscale serve to proxy requests, you can use its identity headers to change what content you serve.
+images: ['/images/id-headers-tailscale-serve-flask/social.png']
+---
+
+If you’ve got multiple users on your tailnet, or if you’ve shared a node out to multiple tailnets, you may want to serve different content based on who’s asking. For personal uses, though, implementing an actual log-in system is cumbersome and tricky to persist across devices.
+
+Under the hood, Tailscale traffic is made up of encrypted WireGuard connections. That means that every connection over your tailnet is authenticated — and that the Tailscale software running on your computer knows which user or device it’s talking to. Since version 1.44, the Tailscale `serve` process now fills this information into headers while proxying requests back to your service.
```

Comment:
Throw this at the end of my previous suggestion?
```suggestion
Since version 1.44, the Tailscale `serve` process now fills this information into headers while proxying requests back to your service.
```